### PR TITLE
Issue #2991: Remove extra {}

### LINKED ISSUE #2991: Extra '{}' exist
### General information

  * Minishift version: 
  * OS: Linux / macOS / Windows
  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox

### Steps to reproduce

  1. Visit https://docs.okd.io/latest/minishift/using/run-against-an-existing-machine.html#running-against-existing-remote-machine , and check the last paragraph.

### Expected
By default, Minishift uses the Allow All identity provider. 

### Actual
By default, {Minishift} uses the Allow All identity provider. 

### Logs

You can start Minishift with `minishift start --show-libmachine-logs -v5` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

### PR DESCRIPTION
**Pull request by liangxia**
Fixes #2991

### CODE REVIEW

File: docs/source/using/run-against-an-existing-machine.adoc [hunk 1]
```diff
@@ -95,6 +95,6 @@ The value of the `--remote-ssh-key` flag must be the location of a private SSH k
 [NOTE]
 ====
 To change the default username and password, you must apply the `htpasswd-identity-provider` add-on with the desired username and password after running the `minishift start` command.
-By default, {{project}} uses the `Allow All` identity provider.
+By default, {project} uses the `Allow All` identity provider.
```
Comment (praveenkumar) [inline docs/source/using/run-against-an-existing-machine.adoc]: :+1:

**Comment by LalatenduMohanty** [comment]
retest this please

**Comment by liangxia** [comment]
Rebased.

**Comment by liangxia** [comment]
retest this please

**Pull request merged**